src/middlewares/isProvided.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-else-return */
+import { Op } from "sequelize";
+import { Ratings } from "../database/models/index";
+import JwtUtil from "../utils/generateToken";
+
+const isProvided = async (req, res, next) => {
+  try {
+    const { id } = req.params;
```

Comment:
> You are repeating the codes we already have. There is a middleware for this.

this is a middleware that checks if the user has already given a feedback so that he doesn't give two feedback on the same product and we don't have that middleware